packages/components/src/components/page-error/page-error.jsx

```diff
@@ -25,16 +25,16 @@ const PageError = ({
 
     return (
         // if image_url is passed we should split the page to two columns and left-align messages
-        <div className={classNames('dc-page-error__container', { 'dc-page-error__container--left': !!image_url })}>
+        <div className={classNames('dc-page-error__container')}>
```

Comment:
No need for `classNames` usage here since only one `className` is defined.

```suggestion
        <div className='dc-page-error__container'>
```

Reply:
Ok, I removed it